README.md

```diff
@@ -1,11 +1,12 @@
 ## Erlang OTP application for handling MaxMind(tm) Geoip2 (.mmdb) files.
 
-This is a forked branch of [brigadier/geodata2](https://github.com/brigadier/geodata2), which uses a shared ets table to hold the database state rather than a cluster of workers. This was done to improve performance.
+This is a forked branch of [altenmueller/geodata2](https://github.com/altenmueller/geodata2), which uses a shared `ETS` table to hold the database state rather than a cluster of workers.  
+This was done to improve performance.
 
 #### Features:
-* Supports all types (hopefully) of bases in .mmdb format, both IPv4 and IPv6 ones;
+* Supports all types (hopefully) of bases in .mmdb format, both IPv4 and IPv6 ones.
 * Safe from binary reference leakage, binary parts are getting copied.
-* R18 compatible
+* R18 compatible.
```

Comment:
It should be OTP22 compatible, at least.

Reply:
Is that ok? I just added:
* OTP22 compatible.
* Uses rebar3.